From 239e49921edf2263495ce2fcd96a01e58d225f72 Mon Sep 17 00:00:00 2001
From: runtime-promotion-bot-noreply
 <runtime-promotion-bot-noreply@jetbrains.com>
Date: Mon, 3 Aug 2026 17:54:09 +0000
Subject: [PATCH] IJI-3336 Updating versions of project dependencies:
 runtimeBuild->25.0.4b329.128

GitOrigin-RevId: 809eec51998fbac985904519c703d8eb6dedb194
---
 build/dependencies/dependencies.properties | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/dependencies/dependencies.properties b/build/dependencies/dependencies.properties
index 4722a117537f..74aac9feef7a 100644
--- a/build/dependencies/dependencies.properties
+++ b/build/dependencies/dependencies.properties
@@ -7,6 +7,6 @@ jdkBuild=21.0.10b1163.108
 launcherBuild=261.19843
 nsisBuild=3.11-1
 restarterBuild=243.6005
-runtimeBuild=25.0.3b329.124
+runtimeBuild=25.0.4b329.128
 snapDockerImage=registry.jetbrains.team/p/ij/docker-hub/jetbrains/snapcraft@sha256:0059496b7a941e27937bc523621a336bddc1327de8dcd03f6b0bd0b57892ce43
 toolboxBuild=3.2.0.65851
-- 
2.53.0

